scripts/eval_k8s.sh

```
@@ -17,5 +17,11 @@ fi
 bash "$PROJECT_ROOT/utils/port_forward.sh"
 
 # Run evaluation
+UNIQUE_ID=$(head /dev/urandom | tr -dc 0-9a-z | head -c 8)
+export UNIQUE_ID
+export TEMP_DIR=$(mktemp -d)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)